addons/godot-firebase/firestore/firestore_task.gd

```diff
@@ -138,7 +138,9 @@ func _on_request_completed(result : int, response_code : int, headers : PoolStri
                 data = null
                 emit_signal("delete_document")
             Task.TASK_QUERY:
-                data = bod
+                data = []
+                for doc in bod:
```

Comment:
What if the result of a query is empty?

Reply:
It should give create_time for request, I added an if check if doc.has('document') so it doesn't error out if query returns no documents